src/components/Pilot/Pilot.jsx

```diff
@@ -0,0 +1,13 @@
+import './Pilot.css'
+
+export const Pilot = ({pilot}) => {
+  const {name, born_at, image_url} = pilot
+
```

Comment:
Você poderia deestruturar diretamente nas props.

```suggestion
export const Pilot = ({name, born_at, image_url}) => {

```